nucypher/characters/lawful.py

```diff
@@ -16,16 +16,23 @@
 """
 
 import json
+import random
+from base64 import b64encode, b64decode
 from base64 import b64decode, b64encode
 from collections import OrderedDict
 from random import shuffle
 
 import maya
 import time
+
+from twisted.python.threadpool import ThreadPool
+
+from bytestring_splitter import BytestringKwargifier, BytestringSplittingError
+from bytestring_splitter import BytestringSplitter, VariableLengthBytestring
 from bytestring_splitter import BytestringKwargifier, BytestringSplitter, BytestringSplittingError, \
     VariableLengthBytestring
```

Comment:
Repeated imports again

Reply:
use an import optimizer!  Perhaps Ill include one in pre-commit.